canonical-web-and-design/ubuntu.com
# Address advantage feedback

## Done
Addressed some of the feedback on the advantage page.

## QA
- Go to /advantage
- See the page looks ok
- Login and check that works
- Check the page looks ok
